src/stores/DocumentationTopicStore.js

```diff
@@ -25,7 +25,6 @@ export default {
   reset() {
     this.state.preferredLanguage = Settings.preferredLanguage;
     this.resetApiChanges();
-    this.resetPageSections();
```

Comment:
This _might_ have been the only place where this method was called. Can it be removed from the base store entirely?